Documentation/project-docs/standard-platform.md

```diff
@@ -23,12 +23,14 @@ The .NET Platform Standard version represents binary portability across platform
 
 
 ## Terms
+- **.NET Platform Standard** - versioned sets of the reference assemblies that all .NET Platforms must support.
```

Comment:
I missed the name change from "Standard Platform" to "Platform Standard".  How am I meant to parse the name?  Is it mean to be "the standard for the .NET Platform"?  If so, the term would seem to be ".NET Platform" rather than ".NET Platform Standard".  Otherwise, I read "Standard" as an edition, e.g. ".NET Platform Standard" vs ".NET Platform Premium" vs ".NET Platform Ultimate", etc.